packages/asc-ui/src/components/Button/Button.stories.tsx

```diff
@@ -43,3 +47,20 @@ storiesOf('Atoms/Buttons', module)
       </ButtonBar>
     </>
   ))
+  .add('with Content', () => (
+    <>
```

Comment:
Why add the fragments?

Reply:
just to make the component beter visible

Reply:
Can you delete this unused fragments?

Reply:
done